src/containers/qr-code/generator/generator-savelink/generator-savedlink.tsx

```diff
@@ -0,0 +1,62 @@
+import { useState, useEffect } from 'react';
+import { Badge } from '@/components/badge';
+
+function QRSavedLinks({
+  qrCodeValue,
+  setQrCodeValue,
+  setIsGenerated,
+}: {
+  qrCodeValue: string;
+  setQrCodeValue: (value: string) => void;
+  setIsGenerated: (value: boolean) => void;
+}): JSX.Element {
```

Comment:
Props 타입은 상단에 따로 선언해주는 걸로 컨벤션 가져가면 좋을 듯하네요!

```suggestion
type Props = {
  qrCodeValue: string;
  setQrCodeValue: (value: string) => void;
  setIsGenerated: (value: boolean) => void;
};

function QRSavedLinks({
  qrCodeValue,
  setQrCodeValue,
  setIsGenerated,
}: Props): JSX.Element {
```